samples/testapp/src/commonMain/kotlin/com/android/identity/testapp/Destinations.kt

```diff
@@ -101,6 +103,21 @@ data object IsoMdocMultiDeviceTestingDestination : Destination {
     override val title = Res.string.iso_mdoc_multi_device_testing_title
 }
 
+data object CertificatesViewerDestination : Destination {
+    override val route = "certificates_viewer"
+    override val title = Res.string.certificates_viewer_title
+}
+
+data object CertificateViewerDestination : Destination {
+    override val route = "certificate_details_viewer"
+    override val title = Res.string.x509_cert_chain_viewer_title
+    const val CERT_NAME = "certificates_name_arg"
+    val routeWithArgs = "$route/{$CERT_NAME}"
+    val arguments = listOf(
+        navArgument(CERT_NAME) { type = NavType.StringType },
+    )
+}
```

Comment:
I don't think it's a good idea to pass a certificate name, please pass the certificate chain itself, encoded as a string like this: `Cbor.encode(certChain.toDataItem()).toBase64Url()`

This way we can later change `AndroidKeystoreSecureAreaScreen` to navigate to the same destination with the "Attestation" button is pressed.